docs/book/src/tasks/experimental-features/cluster-resource-set.md

```diff
@@ -8,3 +8,8 @@ The `ClusterResourceSet` feature is introduced to provide a way to automatically
 
 More details on `ClusterResourceSet` and an example to test it can be found at:
 [ClusterResourceSet CAEP](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20200220-cluster-resource-set.md)
+
+## Update from `ApplyOnce` to `Reconcile`
```

Comment:
I assume we don't want to support a mutable strategy? (or at least not in this PR)

Defintely sounds reasonable to keep it immutable for now.

(no strong opinion either way, just curious)

Reply:
The field was immutable already and given the delete/create pattern should work without side effects, I didn't see a reason to make it mutable. In fact, a user could just create a `Reconcile` CRS with a new name and that will work as well.

But I'm not against making it mutable if there is interest, it would make migrations a bit easier.

Reply:
Sounds good to me.

Let's definitely keep it immutable in this PR. Feel free to open a follow-up issue to see if there's interest or otherwise folks will open an issue over time if there is interest to improve this which is fine as well